am-lib/src/main/java/uk/gov/hmcts/reform/amlib/internal/repositories/AccessManagementRepository.java

```diff
@@ -34,9 +34,7 @@
         + "access_management.resource_id = :resourceId "
         + "and access_management.accessor_id = :accessorId "
         + "and access_management.accessor_type = cast(:accessorType as accessor_type) "
-        + "and access_management.service_name = :resourceDefinition.serviceName "
-        + "and access_management.resource_type = :resourceDefinition.resourceType "
```

Comment:
Can't we do some thing like :relationship is null or access_management.relationship = :relationship for option service_name and resource_type?